tests/test_project.py

```diff
@@ -19,7 +20,6 @@
 from signac.contrib.errors import JobsCorruptedError
 from signac.contrib.errors import WorkspaceError
 from signac.contrib.errors import StatepointParsingError
-from signac.contrib.project import JobsCursor, Project
```

Comment:
Was this accidental? It's causing the tests to fail. I added it back and all tests pass.